articles/users/guides/email-verified.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Email Verified Usage
+description: Outlines proper usage of the email_verified field in a user profile.
+topics:
+    - users
+    - user-management
+    - user-profiles
+contentType:
+    - how-to
+    - reference
+useCase: manage-users
+---
+# Verified Email Usage
+
+The `email_verified` field of a user profile indicates if the user has verified their email address. Valid emails are required for things such as communications to users, password resets/recovery, and passwordless magic links.
+
+An email is usually verified immediately after the user account is created or when the user logs in to the application for the first time. It's a good way to know that the person signing up actually owns the email at that moment. 
+
+Since email verification happens at that specific moment, later on we can't ensure the person logged in with the account still owns the email address. 
+
+Federated identity providers sometimes report if the user has a verified email, and based on that, Auth0 sets the `email_verified` field in the user profile. However, you can't know if the identity provider is doing that properly, or if the verified email from that provider is still owned by the user.
```

Comment:
```suggestion
In case of federated identity providers, they sometimes report if the user has a verified email, and based on that, Auth0 sets the `email_verified` field in the user profile. This, however, transfers the responsibility to the identity provider to do it properly - something we can't ensure. We also don't know if the verified email from that provider is still owned by the user.
```